app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -126,18 +126,41 @@
       @extend .crayons-btn;
     }
 
+    .back-button-container,
+    .next-button {
+      // Explicitly set width to avoid next-button re-sizing with text,
+      // and to ensure that back-button-container takes up the same amount of space.
+      width: $su-10;
+    }
+
     .back-button {
       @extend .crayons-btn;
       @extend .crayons-btn--ghost;
       @extend .crayons-btn--icon;
 
+      $button-size: 40px; // Explicitly set size for accessibility.
+
+      border-radius: 100%;
+      width: $button-size;
```

Comment:
I think that instead of width which is a fixed value, setting this as `min-width: $button-size` may ensure that button is always a minimum size.

Reply:
Ah, I wasn't really thinking that the width was ever going to change/grow for this button, since it has an explicitly-sized `svg` within it, which is why I set it as `width` directly 🤷. But I've gone ahead and changed it to `min-width`.